fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -1227,6 +1233,21 @@ private TransactionSynchronizationRegistry getTrxSynchRegistry() throws FHIRPers
         }
     }
 
+    private List<ExtractedParameterValue> extractCompartmentValues(Resource fhirResource, com.ibm.fhir.persistence.jdbc.dto.Resource resourceDTO) throws Exception {
+
+        List<ExtractedParameterValue> result = null;
+
+        Map<String, Set<java.lang.String>> compartmentRefParams = CompartmentUtil.getCompartmentParamsForResourceType(fhirResource.getClass().getSimpleName());
+
+        if (!compartmentRefParams.isEmpty()) {
+            //result = SearchUtil.extractCompartmentParameterValues(fhirResource, compartmentRefParams);
```

Comment:
Is this meant to be commented out?

Reply:
good question...

Reply:
method wasn't supposed to be there at all. Removed.